src/replica/mutation_log.cpp

```diff
@@ -219,27 +216,30 @@ ::dsn::task_ptr mutation_log_private::append(mutation_ptr &mu,
                                              int hash,
                                              int64_t *pending_size)
 {
-    dassert(nullptr == callback, "callback is not needed in private mutation log");
+    dsn::aio_task_ptr cb =
+        callback ? file::create_aio_task(
+                       callback_code, tracker, std::forward<aio_handler>(callback), hash)
+                 : nullptr;
 
     _plock.lock();
 
+    if (dsn_unlikely(utils::FLAGS_enable_latency_tracer)) {
```

Comment:
no need

Reply:
I don't think so

Reply:
macro contain judge, if you want use dsn_unlikely, you can add in macro, otherwise, it will need judege before `add_point` in anywhere if like this

Reply:
dsn_unlikely is a compiler optimization instruction to improve execution efficiency